packages/cli-kit/src/analytics.ts

```diff
@@ -87,24 +91,37 @@ const buildPayload = async (errorMessage: string | undefined, currentTime: numbe
     }
   }
 
+  const publicPluginData = await fanoutHooks(config, 'public_command_metadata', {})
+  const sensitivePluginData = await fanoutHooks(config, 'sensitive_command_metadata', {})
+
+  const appSpecific = {
+    project_type: await getProjectType(joinPath(directory, 'web')),
+    api_key: appInfo?.appId,
+    partner_id: partnerIdAsInt,
+  }
+
   return {
-    schema_id: 'app_cli3_command/1.0',
-    payload: {
-      project_type: await getProjectType(joinPath(directory, 'web')),
+    public: {
```

Comment:
are we changing the payload schema? is this OK with monorail?  (asking without knowing how monorail events work)

Reply:
It ends up being the same, see `monorail.ts.Schemas`